pkg/apis/pipeline/v1/pipeline_validation.go

```diff
@@ -126,11 +126,19 @@ func (pt PipelineTask) Validate(ctx context.Context) (errs *apis.FieldError) {
 	errs = errs.Also(pt.validateRefOrSpec())
 
 	errs = errs.Also(pt.validateEmbeddedOrType())
-
+	notCustomTaskKinds := map[TaskKind]bool{
+		"":                 true,
+		NamespacedTaskKind: true,
+		"ClusterTask":      true,
```

Comment:
Based on Jerop's comment, "Task" and "ClusterTask" kinds are considered normal tasks if the api version is not set, and customtasks otherwise. so I think the variable name `notCustomTaskKinds` is misleading

Reply:
🤔 what could be a good naming... maybe rename to `taskKinds` and add comment here to explain?

Reply:
I think what you have now looks good!